Bagdasaryan_Inga/Readme.md

```diff
@@ -0,0 +1,56 @@
+# Commence using **Apple Podcasts**
+
+**Apple Podcasts** (known as simply Podcasts in Apple operating systems) is an audio streaming service and media player application developed by Apple Inc. for playing podcasts. Apple began supporting podcasts with iTunes 4.9 released in June 2005 and launched its first standalone mobile app in 2012. **Apple Podcasts** is the best app for finding, following, and listening to millions of the world’s most popular podcasts. You can easily discover new shows through expert curation and personalized recommendations.
+## Requirements
+
+The necessary prerequisites for downloading and utilizing :
++ Apple Podcasts is available on iOS, iPadOS, macOS, watchOS, tvOS, CarPlayand on Amazon Alexa devices.
++ It is essential to have a Wi-Fi connection for utilizing Apple Podcasts.
++ To use all of the features with Apple Podcasts on Alexa, make sure that you update to the latest version of iOS or iPadOS. You should also check the App Store or the Google Play Store to see if there are updates to the Amazon Alexa app.
+
+## Step 1 : Downloading an app
+Apple Podcasts are typically pre-installed on Apple devices. But if you accidentally deleted it, the program can easily be downloaded from Appstore. Just follow this link [Apple Podcasts app](https://apps.apple.com/ua/app/%D0%BF%D0%BE%D0%B4%D0%BA%D0%B0%D1%81%D1%82%D0%B8/id525463029?l=uk).
+
+## Step 2 : Registration on Apple Podcasts
+
+1. In the Podcasts app on your apple device, choose Account => Sign In.
+2. Follow the on-screen instructions to enter your Apple ID and password, then click Sign In. ![](photo.jpeg)
+
+> You can change your Apple ID name, password, or billing information at any time. You can also change the country or region.
+> 1. In the Podcasts app on your Mac, choose Account > View Apple ID to open the Media & Purchases options.
+> 2. Next to your account, click Manage. (You will need to log in).
+> 3. Do any of the following actions :
+>+ Change your Apple ID name or password. Click Edit at "appleid.apple.com".
+>+ Change your payment information. Click Manage Payments.
+>+ Change your billing address. Click Edit.
+>+ Change the country or region code. Click Change country or region.
+## Step 3 : Search and listen to podcasts
+1. In the Podcasts app on your device, click in the search box, then do one of the following:
++ To choose from the available categories, select a topic, such as Comedy or Sports.
++ To search all podcasts, enter a search term in the search box, then select one of the search results. Click "All Podcasts" on the right.
++ To search the media library, enter a search term in the search field, then select one of the search results. Click "Your media library".
+2. In the search results, do one of the following actions:
```

Comment:
The layout becomes a bit overloaded and it is better to divide "Step 2" into two separate sections.